### ISSUE TITLE
Section 2 Test

### ISSUE THREAD

**Issue by linuxtek-canada** [ModernAppsNinja/vspheretanzu101_vt7301#516]
Please check that the information for the questions on this test are in this section.  From what I can see, the majority of the questions are handled in the "vSphere with Tanzu Technical Overview - Elevated View" section, so these questions would belong better in the Chapter 3 test.

**Comment by dglinder** [comment]
Agreed - I re-reviewed the section and can't find why I missed questions #2 and #3 in the section two test.

**Comment by pcgeek2009** [comment]
Personally, I missed 3 the first time and really went back and thought about it. Then I realized the way the question was worded and the answers. I then realized which answer I should have picked, resubmitted, and got the question correct. The way the answers are worded makes it a little tricky, but I was able to answer it correctly.
